Bug: http://github.com/hollie/misterhouse/issues/142
Bug: http://github.com/hollie/misterhouse/issues/141
Author: Robert James Clay <jame@rocasa.us>
Last-Update: 20130325
Description: Correct path in Perl script shabang lines.

--- a/bin/display_callers
+++ b/bin/display_callers
@@ -1,5 +1,3 @@
-#!d:/perl/bin/perl.exe
-# -*- Perl -*-
 #!/usr/bin/perl
 #--------------------------------------------------------------------------- 
 #  File: 
--- a/bin/test_x10
+++ b/bin/test_x10
@@ -1,4 +1,4 @@
-#!/bin/perl -w
+#!/usr/bin/perl -w
 
 # Use this as a stand-alone (outside of mh) way to
 # test a CM11 and/or CM17 X10  device
--- a/bin/xAP-festival.pl
+++ b/bin/xAP-festival.pl
@@ -1,4 +1,4 @@
-#!/bin/perl
+#!/usr/bin/perl
 
 =begin comment
 
---
--- a/web/bin/dbmedit.cgi
+++ b/web/bin/dbmedit.cgi
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 #   DBMEdit 1.0
 #
--- a/web/bin/weather_graph_zoom.pl
+++ b/web/bin/weather_graph_zoom.pl
@@ -1,4 +1,4 @@
-#!/local/bin/perl
+#!/usr/bin/perl
 #
 # $Date$
 # $Revision$
--- a/web/bin/test_cgi_error.pl
+++ b/web/bin/test_cgi_error.pl
@@ -1,4 +1,4 @@
-#!/local/bin/perl 
+#!/usr/bin/perl 
 
 # Test for returning errors
 
--- a/web/newclock/index.pl
+++ b/web/newclock/index.pl
@@ -1,4 +1,4 @@
-#!/local/bin/perl
+#!/usr/bin/perl
 
 
 use CGI;
--- a/web/newclock/alarm.pl
+++ b/web/newclock/alarm.pl
@@ -1,4 +1,4 @@
-#!/local/bin/perl
+#!/usr/bin/perl
 
 
 use CGI;
--- a/web/organizer/calendar.pl
+++ b/web/organizer/calendar.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # $Date$
 # $Revision$
--- a/web/organizer/contacts.pl
+++ b/web/organizer/contacts.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # ----------------------------------------------------------------------------
 # contacts.pl
 # Copyright (c) 2001 Jason M. Hinkle. All rights reserved. This script is
--- a/web/organizer/tasks.pl
+++ b/web/organizer/tasks.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # ----------------------------------------------------------------------------
 # tasks.pl
 # Copyright (c) 2001 Jason M. Hinkle. All rights reserved. This script is
--
1.7.10.4
